src/main/java/frc/robot/subsystems/BuddyBalanceSubsystem.java

```diff
@@ -4,107 +4,187 @@
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.hardware.AbsoluteEncoder;
 import frc.robot.hardware.MotorController;
+import frc.robot.hardware.AbsoluteEncoder.EncoderConfig;
 import frc.robot.hardware.MotorController.MotorConfig;
 import frc.robot.classes.TunableNumber;
 
 import com.revrobotics.CANSparkMax;
-import com.revrobotics.RelativeEncoder;
-import com.revrobotics.SparkMaxPIDController;
 
 public class BuddyBalanceSubsystem extends SubsystemBase {
-  private static double kBalancedPosition; // Buddy balance PID reference point when lifting a robot and engaging charge station
-  private static double kDeployedPosition; // Buddy balance PID reference point when setting down a robot/initial position when deployed
-  // TODO: Make the reference point constants and default motor PID values final when they are done being tuned with TunableNumbers
-  private static final double kDefaultMotorP = 1e-6;
+  private static final double kRetrievedAngle = 180; // Buddy balance PID reference point when lifting a robot and engaging charge station
+  private static final double kDeployedAngle = 0; // Buddy balance PID reference point when setting down a robot/initial position when deployed
+  // TODO: Make the constants final when they are done being tuned with TunableNumbers
+  private static final double kDefaultMotorP = 0.0001;
   private static final double kDefaultMotorI = 0;
-  private static final double kDefaultMotorD = 1e-6;
-  private static final int deployServoID = 1;
-  private static double kServoDeployedPos = 1;
+  private static final double kDefaultMotorD = 0;
+  private static final int deployServo1ID = 8;
+  private static final int deployServo2ID = 9;
+  private static final double kServoInitialPos = 0.5;
+  private static final double kServoDeployedPos1 = 0;
+  private static final double kServoDeployedPos2 = 1;
+  private double tunedRetrievedAngle;
+  private double tunedDeployedAngle;
+  private double tunedServoDeployedPos1;
+  private double tunedServoDeployedPos2;
 
   private TunableNumber refPointBalancedTuner;
   private TunableNumber refPointDeployedTuner;
-  private TunableNumber refPointServoTuner;
-  private TunableNumber tunerNumRightP;
-  private TunableNumber tunerNumRightI;
-  private TunableNumber tunerNumRightD;
-  private TunableNumber tunerNumLeftP;
-  private TunableNumber tunerNumLeftI;
-  private TunableNumber tunerNumLeftD;
-
-  private Servo activateDeploy;
-  private CANSparkMax rightMotor;
-  private CANSparkMax leftMotor;
-  private SparkMaxPIDController rightPIDController;
-  private SparkMaxPIDController leftPIDController;
-  private RelativeEncoder rightEncoder;
-  private RelativeEncoder leftEncoder;
+  private TunableNumber refPointServoTuner1;
+  private TunableNumber refPointServoTuner2;
+  private TunableNumber tunerNumP;
+  private TunableNumber tunerNumI;
+  private TunableNumber tunerNumD;
```

Comment:
Are these Tunable numbers in use?  Let's clean them up if not in use

Reply:
They are being used